app/main.py

```diff
@@ -2,0 +1,41 @@
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared among all instances of `Animal`. If the intention is to track all living animals across all instances, this is fine. However, if each instance should have its own list of 'alive' animals, consider making it an instance variable instead.